src/test/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinterTest.java

```diff
@@ -66,8 +71,31 @@ public void testParseFileWithError() throws Exception {
             final String expected = "[ERROR:0] Javadoc comment at column 1 has parse error. "
                     + "Missed HTML close tag 'qwe'. Sometimes it means that close tag missed "
                     + "for one of previous tags.";
-            Assert.assertEquals(expected, ex.getMessage());
+            assertEquals(expected, ex.getMessage());
         }
     }
 
+    @Test
+    public void testCreationOfFakeCommentBlock() throws Exception {
+        final Method createFakeBlockComment =
+                Whitebox.getMethod(DetailNodeTreeStringPrinter.class,
+                        "createFakeBlockComment", String.class);
```

Comment:
@rnveach I had tried, but I didn't manage...  If you take a look at this method in the [gist](https://nimfadora.github.io/com.puppycrawl.tools.checkstyle.treewalker/DetailNodeTreeStringPrinter.java.html) you'll see several survived mutation. The thing is that any code that uses this method does not rely on any values which being set on "red" lines. And the output of this method is incapsulated only in local variables. So, I didn't find a way to get it without reflection.
From some perspective, those lines could be successfully removed. But they provide a proper structure of DetailAST object which corresponds to well formatted javadoc, so it may be useful in future.